bigchaindb/events.py

```diff
@@ -1,33 +1,91 @@
-from enum import Enum
+from collections import defaultdict
 from multiprocessing import Queue
 
 
-class EventTypes(Enum):
+POISON_PILL = 'POISON_PILL'
+
+
+class EventTypes:
+    """Container class that holds all the possible
+    events BigchainDB manages."""
+
+    # If you add a new Event Type, make sure to add it
+    # to the docs in docs/server/source/event-plugin-api.rst
+    ALL = ~0
```

Comment:
This is a *bit* (eh eh eh) tricky. It's the bitwise negation of `0`, that is a number with all `1`s. It's always `True` as long as you `&` with something `True`:

```
>>> bin(~0 & 0b0010000)
'0b10000'
>>> bin(~0 & 0b0)
'0b0'
>>> bin(~0 & 0b0000001)
```